.github/workflows/porch-e2e.yml

```diff
@@ -0,0 +1,78 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Porch End-to-End Tests
+on:
+  push:
+    paths-ignore:
+      - "docs/**"
+      - "site/**"
```

Comment:
May be we can trigger this on changes in porch codebase ?

Reply:
I'll look to restrict more; we shouldn't do 'just porch' because of dependencies on kpt libraries (i.e. renderer) that can cause Porch to fail if bugs are introduced.

Reply:
done; ptal. I'd be uncomfortable restricting more of the kpt code at this point. We can evaluate if we see that this test is causing a bottleneck

Reply:
hmm..you are right, we should use `paths` instead of `paths-ignore` ([gh workflow docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore) to trigger on selected paths (porch/, kpt/internal/render...) ?

Reply:
I don't think we should use the inverse logic because it will make it harder to update. I'd like to (at least while we have high velocity development) err on the side of automatically running tests when we add directories etc. than missing adding paths to the list :(

Reply:
That's a fair point.